tests/cosmosLightClient/test-nim-light-client-in-cosmos.ts

```diff
@@ -273,7 +309,9 @@ describe('Light Client In Cosmos', () => {
     const headerSlotAfterAllUpdates = byteArrayToNumber(
       queryResultAfterAllUpdates.slice(0, 8),
     );
-
+    fs.writeFileSync('gasLightClient', JSON.stringify(gasArrayLightClient), {
```

Comment:
Please use a better name for the file
` fs.writeFileSync('gasLightClient.json', JSON.stringify(gasArrayLightClient), {`